src/features/groups/components/GroupSingleHeaderDropdownMenuTriggerButton/GroupSingleHeaderDropdownMenuContent/index.tsx

```diff
@@ -37,7 +37,7 @@ export const GroupSingleHeaderDropdownMenuContent = ({
   };
 
   return (
-    <DropdownMenuContent>
+    <DropdownMenuContent alignOffset={16}>
```

Comment:
Could you please add a comment that explains this offset size must match the size of theside padding of the body?